.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
```

Comment:
Let's do this in a separate PR.

Reply:
Disolved to https://github.com/jonhoo/fantoccini/pull/144